projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+#define DELAY_TIME_MS 500
+
+typedef struct counters {
```

Comment:
In `typedef struct counters`, `counters` should be capitalized

Reply:
Thanks Avery, I have fixed this as well.